packages/app/components/token-picker/TokenPickerCommonTokens.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+
+import { RadioButton } from "@/ui";
+import { TokenIcon } from "../TokenIcon";
+import { Token } from "@/models/token/types";
+
+const COMMON_TOKENS = [
+  {
+    address: "0x532801ED6f82FFfD2DAB70A19fC2d7B2772C4f4b",
+    name: "Swapr on Gnosis chain",
+    symbol: "SWPR",
+    decimals: 18,
+    logoURI: "/assets/images/tokens/swapr.png"
+  },
+  {
+    address: "0x6A023CCd1ff6F2045C3309768eAd9E68F978f6e1",
+    name: "Wrapped Ether on Gnosis chain",
+    symbol: "WETH",
+    decimals: 18,
+    logoURI: "/assets/images/tokens/weth.png"
+  },
+  {
+    address: "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d",
+    name: "Wrapped XDAI",
+    symbol: "WXDAI",
+    decimals: 18,
+    logoURI: "/assets/images/tokens/wxdai.png"
+  },
+  {
+    address: "0x9C58BAcC331c9aa871AFD802DB6379a98e80CEdb",
+    name: "Gnosis Token on Gnosis chain",
+    symbol: "GNO",
+    decimals: 18,
+    logoURI: "/assets/images/tokens/gno.png"
+  }
+];
+
+export const TokenPickerCommonTokens = () => (
+  <div className="mt-5 px-6">
+    <p className="text-xs font-semibold text-em-low">Common tokens</p>
+    <div className="flex flex-wrap mt-2 space-x-2 space-y-2">
```

Comment:
We can change the space y/x 2 with gap (applies both) 

```suggestion
    <div className="flex flex-wrap mt-2 gap-2">
```
and also it does not  add space to the beginning when it wraps, fixing this:

<img width="280" alt="image" src="https://github.com/SwaprHQ/stackly-ui/assets/5664434/015b7b18-a56a-40ee-8a79-b6746695b350">

after:
<img width="293" alt="image" src="https://github.com/SwaprHQ/stackly-ui/assets/5664434/f6f54230-22aa-4a1d-9f4c-b5ac841cf216">

Reply:
Oh I like this one better, thanks!